redhat-cloud-client-configuration.spec

```diff
@@ -64,22 +64,44 @@ install -D -m644 rhcd-stop.service %{buildroot}%{_unitdir}/
 
 # Make sure that rhsmcertd.service is enabled and running
 %systemd_post rhsmcertd.service
+# Run following block only during installation (not during update)
 if [ $1 -eq 1 ]; then
     # Try to get current value of auto-registration in rhsm.conf
     subscription-manager config --list | grep -q '^[ \t]*auto_registration[ \t]*=[ \t]*1'
     if [ $? -eq 0 ]; then
-        auto_reg_already_enabled=1
+        auto_reg_enabled=1
     else
-        auto_reg_already_enabled=0
+        auto_reg_enabled=0
     fi
-    if [ $auto_reg_already_enabled -eq 0 ]; then
-        # Save original rhsm.conf
+
+    # Try to get current value of manage_repos
+    subscription-manager config --list | grep -q '^[ \t]*manage_repos[ \t]*=[ \t]*0'
+    if [ $? -eq 0 ]; then
```

Comment:
There's a cleaner way to do this pattern of "run a command; check its exit code" that I found using shellcheck. It's documented in [SC2181](https://www.shellcheck.net/wiki/SC2181).

```bash
if subscription-manager config --list | grep [...]; then
    manage_repos_enabled=1
fi
```

Could this even be taken a step further and assign the value of `manage_repos_enabled` to the exit code of grep?

Reply:
Fair enough. I'm fine with either approach.